persist/fs/commitlog/options.go

```diff
@@ -78,6 +82,23 @@ func NewOptions() Options {
 	return o
 }
 
+func defaultRetentionOptions() retention.Options {
+	return retention.NewOptions().
+		SetBlockSize(15 * time.Minute)
+}
+
+func (o *options) Validate() error {
+	if o.FlushInterval() < 0 {
```

Comment:
Might be best to actually make sure it's greater than zero.  Needs to be some > 0 value or else we'll never tick.